sql/wsrep_mysqld.cc

```diff
@@ -2412,8 +2446,15 @@ void wsrep_to_isolation_end(THD *thd)
     wsrep_cleanup_transaction(thd);
   }
 
-  if (thd->wsrep_skip_wsrep_hton)
+  if (thd->wsrep_skip_wsrep_hton) {
+    DEBUG_SYNC(thd, "wsrep_to_isolation_end_before");
```

Comment:
Likewise, I would suggest renaming the sync points to `wsrep_TOI_end_before_wsrep_skip_hton` and `wsrep_TOI_begin_after_wsrep_skip_hton`.

Reply:
agree, name changed